docs/source/API/core/view/realloc.rst

```diff
@@ -0,0 +1,156 @@
+``realloc``
+===========
+
+Header File: ``<Kokkos_Core.hpp>``
+
+.. role:: cppkokkos(code)
+    :language: cppkokkos
+
```

Comment:
Please move towards a standardized structure for API documentation entries:  function name, Header File, (human language) Description, Usage and Example.